src/core/core.scale.js

```diff
@@ -569,6 +569,60 @@ module.exports = function(Chart) {
 				0;
 		},
 
+		/**
+		 * Returns a subset of ticks to be plotted.
+		 * This functionality makes it so that we don't have overlapping labels.
+		 */
+		autoSkip: function() {
```

Comment:
I had debated making `ticks` a parameter, so I've changed that since you've suggested it as well

I had also though about making it private, but I think that it's a method that users are potentially likely to want to override with their own implementation

I'm not a big fan of `disseminate` as a name. We're not really spreading the ticks, but rather are skipping some